lib/delta.ex

```diff
@@ -492,4 +492,102 @@ defmodule Delta do
         inverted
     end
   end
+
+  @doc ~S"""
+  Returns a delta representing the difference between two documents.
```

Comment:
Can you clarify what you mean here? The diff does account for changes in embeds, though at a very coarse level. But changing the content of an embed will result in a delete and an insert to represent that change.

Reply:
Oh sorry, I meant that changes in embeds won't be diffed. `Delta.compose` [for example](https://github.com/slab/delta-elixir/blob/6c20896ff868d79f7430413efeb6d34d48d762e1/lib/delta/op.ex#L282-L285) calls a custom embed's `compose` callback if it encounters one. By default I would expect `diff` to do the same. Though feel free to ignore this if it doesn't make sense, it might be just me 😅

Reply:
Ah, I get ya now. While I think the current behavior makes sense, because there isn't a way in a delta to represent changes within an embed, I'm happy to note it in the docstring for clarity.